src/Components/Heading.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <header className="App-header">
+      <img
+        className="img-logo"
+        src="https://image.flaticon.com/icons/svg/139/139899.svg"
+        alt="Hotel Logo"
+      />
+      CYF Hotel Alex
```

Comment:
Do you think we could receive this text via props?

Reply:
<Heading logoName={"CYF Hotel Alex"} />

Reply:
I don’t think logoName is the best name as possible. How about displayText or label only?